src/main/webapp/app/entities/exercise-hint/exercise-hint-update.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="row justify-content-center">
+    <div class="col-8">
+        <form name="editForm" role="form" novalidate (ngSubmit)="save()" #hintForm="ngForm">
+            <h2 id="jhi-exercise-hint-heading" jhiTranslate="artemisApp.exerciseHint.home.createOrEditLabel">Create or edit a Exercise Hint</h2>
+            <div>
+                <jhi-alert-error></jhi-alert-error>
+                <div class="form-group" [hidden]="!exerciseHint?.id">
+                    <label for="id" jhiTranslate="global.field.id">ID</label>
+                    <input type="text" class="form-control" id="id" name="id" readonly [(ngModel)]="exerciseHint.id" />
+                </div>
+                <div class="form-group">
+                    <label class="form-control-label" jhiTranslate="artemisApp.exerciseHint.title" for="field_title">Title</label>
+                    <input type="text" required minlength="5" class="form-control" name="title" id="field_title" [(ngModel)]="exerciseHint.title" />
+                </div>
+                <div class="form-group hint-form__editor-wrapper">
+                    <jhi-markdown-editor
+                        [markdown]="exerciseHint.content"
+                        (markdownChange)="updateHintContent($event)"
+                        [resizableMinHeight]="MarkdownEditorHeight.SMALL"
+                        [resizableMaxHeight]="MarkdownEditorHeight.LARGE"
+                        [enableResize]="true"
+                        [domainCommands]="domainCommands"
+                        [editorMode]="editorMode"
+                    ></jhi-markdown-editor>
+                </div>
+
+                <div class="form-group">
+                    <label class="form-control-label" jhiTranslate="artemisApp.exerciseHint.exercise" for="field_exercise">Exercise</label>
+                    <select class="form-control" id="field_exercise" name="exercise" [(ngModel)]="exerciseHint.exercise" [attr.disabled]="true">
```

Comment:
Are you sure we need a select here? Because I only have one exercise and as far as I understand our mapping is one to one.

Reply:
This was part of the jhipster boilerplate and is documented in the component: Atm it is not possible to move hints between exercises but it could be a feature in future. This is why I did not remove this part but just disabled the dropdown.

Reply:
I would still suggest to replace it remove the select. When we will have it for multiple exercises in the future, then we can change it back to select

Reply:
I changed it to an input.